test/e2e/cypress/e2e/about.cy.js

```diff
@@ -2,7 +2,7 @@ import { getValue } from '../support/common';
 
 describe('User account page', () => {
   before(() => {
-    cy.navigateToItem('About');
+    cy.visit('/about');
     cy.url().should('include', '/about');
```

Comment:
Is this assertion `cy.url().should('include', '/about');` needed when we do `cy.visit('/about');`

Reply:
I was thinking about that, but since we have some route guards that can change the route url, I just added as a defensive movement to debug the e2e tests in case of failing, but we can remove